app/services/import_companies.rb

```diff
@@ -39,6 +39,27 @@ def header_converter
     end
   end
 
+  def create_mq_assessment!(row, company)
+    MQ::Assessment.create!(
+      mq_assessment_attributes(row).merge(company_id: company.id)
+    )
+  end
+
+  def create_cp_assessment!(row, company)
+    assumptions = row[:assumptions]&.strip
+    cp_assessment_date = row[:carbon_performance_assessment_date]&.strip&.downcase
+
+    return if cp_assessment_date == 'not assessed' && assumptions.blank?
```

Comment:
I would extract method:
```rb
def create_cp_assessment!(row, company)
  return if assessment_row_invalid?(row) # not sure about the name
  ..
```

Reply:
hmm, actually I'm striping and downcasing values and using them to create assessment later so I wouldn't move that into a separate method to not do it again.

Reply:
ok, that's fine then